django/db/migrations/writer.py

```diff
@@ -195,9 +196,12 @@ def as_string(self):
         if self.migration.replaces:
             items['replaces_str'] = "\n    replaces = %s\n" % self.serialize(self.migration.replaces)[0]
         # Hinting that goes into comment
+        version_and_timestamp = ""
+        if not self.no_header:
```

Comment:
That's a double negation the writer shouldn't have to deal with as it's a side effect of the fact the CLI facts are negated.